### DISCUSSION TITLE
Hacktoberfest shout-out

### DISCUSSION THREAD

**Discussion by PyvesB** [badges/shields#5789] [Show and tell] [answered]
@MonliH, @RDIL, @outsideris, @hjagodzinski, @JoeIzzard, @jabbar86 and @ThakurKarthik you've all contributed to Shields.io during Hacktoberfest either through #5555 or other issues. Thanks for all your valuable contributions!

If you've got a Twitter handle, please share it here, I'll post a small public shout-out to you folks. 😉 

Do let me know if I've left anyone out!

**Comment by jabbar86** [comment]
Hello, @PyvesB here is my Twitter handle `memonjabbar1`. It was a pleasure working with you guys.

**Comment by ThakurKarthik** [comment]
Hey @PyvesB thanks for the oppurtunity, My twitter handle is Karthik12778390

**Comment by RDIL** [comment]
Hey @PyvesB, mine is `@rdil_pickle` :)

**Comment by hjagodzinski** [comment]
Hi guys, mine is `@hjagodzinski`. Thanks for the opportunity to be a part of Hacktoberfest! It wouldn't be possible without you guys.

**Comment by JoeIzzard** [comment]
Hey @PyvesB, not much of a twitter user but thanks for the shout-out here! Although it seems Shields has been determined an invalid repo for me 👎

**Reply by PyvesB** [reply]
That's super odd! We did have the `hacktoberfest` topic on the repo during the entirety of October!

**Reply by PyvesB** [reply]
Could you please a screenshot of the page, I would be interested to see what exactly they're saying!

**Reply by calebcartwright** [reply]
@JoeIzzard - when did you check? the topic was removed but added back recently

https://github.com/badges/shields/pull/5772#issuecomment-720392719

**Reply by PyvesB** [reply]
To be completely transparent, the person who removed the topic was probably me. I did a pass through all my repos to remove it the other day. When I saw the topic still on Shields.io this evening, I thought I had missed this specific repo. But in light of @calebcartwright's message, I probably did remove it here as well and he added it back.

Sorry about this. It was an honest mistake, I thought it was no longer needed as Hacktoberfest is finished, but apparently it's still used for counting post-October. Hopefully if you check back again @JoeIzzard, things will have sorted themselves out :)

**Reply by JoeIzzard** [reply]
@PyvesB Screenshot attached.

![Screenshot 2020-11-04 at 9 00 59 pm](https://user-images.githubusercontent.com/3457283/98167428-f6f9b680-1ee0-11eb-902c-ef9122dcce43.png)

And this is the explanation on the site.

![Screenshot 2020-11-04 at 10 34 54 pm](https://user-images.githubusercontent.com/3457283/98175193-03384080-1eee-11eb-861d-02d1e83996ea.png)

It would seem to be the same as noted in that issue @calebcartwright.

**Reply by JoeIzzard** [reply]
No worries @PyvesB, I will check back in a bit and see if anything changes.

**Reply by PyvesB** [reply]
I think the Hacktoberfest system is getting confused. The two ineligible pull requests were submitted on 16 October, and the fourteen day review period brings us to October 30th. The `hacktoberfest` topic was definitely present during that entire period, and the two other pull requests you submitted on the same day were fine.

I've added the `hacktoberfest-accepted` label to the two problematic pull requests, maybe that will unblock the system!

**Reply by chris48s** [reply]
This is a reported issue at https://github.com/digitalocean/hacktoberfest/issues/724
My reading is that adding the `hacktoberfest-accepted` label won't change anything and DO don't plan to address it.

**Reply by PyvesB** [reply]
That linked issue suggests that all accepted PRs would have been nuked. Only half of @JoeIzzard's ones seem to have been affected though.

Either way, such a shame that they don't plan to address it.

**Reply by calebcartwright** [reply]
I feel like things with Hacktoberfest were more confusing this year than in years past. I know the team has a lot to deal with, especially with the ever increasing spam, but hoping things will be more smooth next year.

**Reply by JoeIzzard** [reply]
I hope so, first year I have actually had time to participate thanks to covid and there "logic flaw" as they put it kinda puts new participants like me off it to be honest.

**Comment by outsideris** [comment]
My tw handle is `outsideris`. Thank you.

**Comment by PyvesB** [comment, marked as answer]
As promised, here is the little Tweet: https://twitter.com/Shields_io/status/1324469521818783751

If some people have been accidentally left out, let me know, I can easily tag you in a response! 😉
